extension/packages/nextjs/app/eip-712/page.tsx

```diff
@@ -103,14 +108,27 @@ const Eip712: NextPage = () => {
   return (
     <div className="flex items-center flex-col flex-grow pt-10 px-8">
       <div className="flex flex-col gap-4 items-center">
+        <div className="text-center">
+          <h1 className="text-2xl font-bold">EIP-712</h1>
+          <div className="max-w-2xl">
+            EIP-712 defines a standard for hashing and signing typed structured
+            data in Ethereum. It enhances security and usability by enabling the
+            creation of more readable and secure signed messages, reducing the
+            risk of phishing attacks and user errors. For more details, visit
+            the{" "}
+            <Link
```

Comment:
Ohh since this is an external link maybe its nice to have it as plain html `a` tag instead of using Next Link ? 

Also maybe we could have `target="_blank"` ?  So that it opens in new page ?

Reply:
[b8596fc](https://github.com/scaffold-eth/create-eth-extensions/pull/5/commits/b8596fc97e438f96260861169e157e8a1bb58c48)